tests/testthat/test-convert_na_to.R

```diff
@@ -77,7 +77,7 @@ test_that("convert_na_to - numeric: arg 'replacement' must be of length one", {
 
 test_that("convert_na_to - character: returns original vector if 'replacement' not good", {
   expect_equal(
-    convert_na_to(c("a", "b", "c", NA), replacement = 1, verbose = FALSE),
+    convert_na_to(c("a", "b", "c", NA), replacement = mtcars, verbose = FALSE),
     c("a", "b", "c", NA)
```

Comment:
Can you also test against the expected input (i.e. a numeric vector)?

Reply:
Done, thanks for pointing that out